docs/content/users/quickstart.md

```diff
@@ -164,7 +164,7 @@ Start a new [Craft CMS](https://craftcms.com) project or retrofit an existing on
 !!!tip "Compatibility with Craft CMS 3"
     The `craftcms` project type is best with Craft CMS 4+, which is more opinionated about some settings. If you are using Craft CMS 3 or earlier, you may want to use the `php` project type and [manage settings yourself](https://github.com/ddev/ddev/issues/4650).
 
-Environment variables will be automatically added to your `.env` file to simplify the first boot of a project. For _new_ installations, this means the default URL and database connection settings displayed during installation can be used without modification. If _existing_ projects expect environment variables to be named in a particular way, you are welcome to rename them.
```

Comment:
Since the `.ddev/.env.web` is always created, the distinction between new/existing doesn't really matter. `CRAFT_DB_` variables will always override.